### PR TITLE
created new POM class

### PR DESCRIPTION
**Pull request by BhimashankarS**
Please review

### CODE REVIEW

File: ComcastCRMFrameworkSample/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.comcast.crm.objectrepositoryutility;
+
+public class CreateProductPage {
```
Comment (madhura-1992) [inline ComcastCRMFrameworkSample/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:3]: change the xpath
do not hardcode the data

**Review by madhura-1992 — COMMENTED** [review]
added few comments

File: ComcastCRMFrameworkSample/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java (hunk 1 above)
Comment (BhimashankarS) [reply ComcastCRMFrameworkSample/src/main/java/com/comcast/crm/objectrepositoryutility/CreateProductPage.java:3]: fixed

**Comment by madhura-1992** [comment]
merged

**Pull request merged**